src/parser/toplevel.lisp

```diff
@@ -1048,6 +1050,11 @@ Ensure the package exists, import the COALTON package and required dependencies,
        (push instance (program-instances program))
        t))
 
+    ((coalton:lisp-toplevel)
+     (loop :for form :in (cst:raw (cst:rest form))
+           :do (vector-push-extend form (program-lisp program)))
```

Comment:
Either push to a list or update the other places in this file to use a vector.

Reply:
This uses a list now.